layers/+frameworks/vue/packages.el

```diff
@@ -0,0 +1,58 @@
+;;; packages.el --- react layer packages file for Spacemacs. -*- lexical-binding: t -*-
+;;
+;; Copyright (c) 2012-2019 Sylvain Benner & Contributors
+;;
+;; Author: Thanh Vuong <thanhvg@gmail.com>
+;; URL: https://github.com/thanhvg
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+
+(defconst vue-packages
+  '(web-mode
+    add-node-modules-path
+    company
+    emmet-mode
+    evil-matchit
+    flycheck
+    prettier-js
+    smartparens
+    yasnippet))
+
+(defun vue/post-init-web-mode ()
+  (define-derived-mode vue-mode web-mode "Vue")
+  (add-to-list 'auto-mode-alist '("\\.vue\\'" . vue-mode))
+  (spacemacs/add-to-hook 'vue-mode-hook '(spacemacs//vue-setup-editor-style
+                                          spacemacs//vue-setup-keybindings))
+  (add-hook 'vue-mode-local-vars-hook #'spacemacs//vue-setup-backend))
+
+(defun vue/post-init-add-node-modules-path ()
+  (add-hook 'vue-mode-hook #'add-node-modules-path))
+
+(defun vue/post-init-company ()
+  (add-hook 'vue-mode-local-vars-hook #'spacemacs//vue-setup-company))
+
+(defun vue/post-init-emmet-mode ()
+  (add-hook 'vue-mode-hook #'emmet-mode))
```

Comment:
`emmet-mode` comes with `web-mode`, so I think this is unnecessary.

Reply:
yeah you're right, this is not needed